package.json

```diff
@@ -44,10 +44,13 @@
     "bip39": "^2.5.0",
     "ipfs": "^0.32.2",
     "ipfs-api": "^24.0.0",
+    "ipfs-postmsg-proxy": "^3.1.1",
     "ipfs-pubsub-room": "^1.4.0",
     "js-sha256": "^0.9.0",
     "muport-core": "^0.4.1",
     "orbit-db": "^0.19.9",
+    "orbit-db-cache": "^0.2.4",
```

Comment:
Why do we need this? Looking at the code in 3box.js we could just use the default in orbit by not passing an alternative cache?

Reply:
ah yup, good catch, this was before I made the pr to orbitdb and I had different implementation

Reply:
ok :)
Unrelated but I'm pretty sure we can remove the `ipfs-pubsub-room` dep.